storage/cloud-client/acl.py

```diff
@@ -0,0 +1,255 @@
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to manage access control lists (acls) in
+Google Cloud Storage.
+
+For more information, see the README.md under /storage and the documentation
+at https://cloud.google.com/storage/docs/encryption.
+"""
+
+import argparse
+
+from gcloud import storage
+
+
+def get_bucket_acl(bucket_name):
+    """Prints out a bucket's access control list."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    for entry in bucket.acl:
+        print('{}: {}'.format(entry['role'], entry['entity']))
+
+
+def get_bucket_acl_for_user(bucket_name, user_email):
+    """Prints out a bucket's access control list for a given user."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    bucket.acl.reload()
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # get the roles for different types of entities.
+    roles = bucket.acl.user(user_email).get_roles()
+
+    print(roles)
+
+
+def set_bucket_acl(bucket_name, user_email):
+    """Adds a user as an owner on the given bucket."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # grant access to different types of entities. You can also use
+    # `grant_read` or `grant_write` to grant different roles.
+    bucket.acl.user(user_email).grant_owner()
+    bucket.acl.save()
+
+    print('Added user {} as an owner on bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def remove_bucket_acl(bucket_name, user_email):
+    """Removes a user from the access control list of the given bucket."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # remove access for different types of entities.
+    bucket.acl.user(user_email).revoke_read()
+    bucket.acl.user(user_email).revoke_write()
+    bucket.acl.user(user_email).revoke_owner()
+    bucket.acl.save()
+
+    print('Removed user {} from bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def set_bucket_default_acl(bucket_name, user_email):
+    """Adds a user as an owner in the given bucket's default object access
+    control list."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # grant access to different types of entities. You can also use
```

Comment:
A newline between sentences would make it easier to scan, since the sentences are about different subjects.